src/sentry/api/endpoints/release_files.py

```diff
@@ -148,6 +149,9 @@ def post(self, request, project, version):
         if not full_name:
             return Response({'detail': 'File name must be specified'}, status=400)
         name = full_name.rsplit('/', 1)[-1]
+        pattern = re.compile(r"[\n\t\r\f\v\\]")
```

Comment:
can you compile the re as a global? like _fn_re = re.compile(...) on module level and then use it from here. The regex compilation in Python is really not very fast.